test/human_player_test.exs

```diff
@@ -0,0 +1,11 @@
+defmodule HumanPlayerTest do
+  use ExUnit.Case
+
+  import ExUnit.CaptureIO
+
+  test "returns a position on the board" do
+    assert capture_io([input: "1\n"], fn ->
+      assert HumanPlayer.choose_position("X") == 1
```

Comment:
Why does "X" matter in this test? I don't see you either asserting on something with X or it being connected to anything else...